statsmodels/discrete/discrete_model.py

```diff
@@ -747,21 +747,20 @@ def predict(self, params, exog=None, exposure=None, offset=None,
         #TODO: add offset tp
         if exog is None:
             exog = self.exog
-            offset = getattr(self, 'offset', 0)
+        
+        if exposure is None:
             exposure = getattr(self, 'exposure', 0)
```

Comment:
just as a reminder: you could add a code comment `# self.exposure is already in log`